mlcube/vision_data_selection/README.md

```diff
@@ -0,0 +1,210 @@
+# `dataperf-vision-selection`: A Data-Centric Visual Benchmark for Training Data Selection
```

Comment:
We should change this readme for the one MLCube Vision Dataperf tutorial, as most of the information contained here is now outdated or replaced by MLCubes functionality.